src/Entities/RulesEngine/Bind.php

```diff
@@ -0,0 +1,172 @@
+<?php
+/**
+ * This source file is proprietary and part of Rebilly.
+ *
+ * (c) Rebilly SRL
+ *     Rebilly Ltd.
+ *     Rebilly Inc.
+ *
+ * @see https://www.rebilly.com
+ */
+
+namespace Rebilly\Entities\RulesEngine;
+
+use DomainException;
+use Rebilly\Rest\Resource;
+
+/**
+ * Class Bind.
```

Comment:
wrong description

Reply:
Why?

Reply:
Mmmm, actually I don't know why I left this message here, maybe I missread classname or this message was for another file, sorry